### PR TITLE
Remove duplicate bug ids in source code.

### LINKED ISSUE #23140: ManagedHandler: ManagedHandler_DefaultCredentialsTest tests are disabled
These tests are disabled for the managed handler.  They need to be investigated and either src or tests fixed.

### PR DESCRIPTION
**Pull request by Priya91**
fixes #23140 

These tests will anyways be skipped by failing the conditional theory on each individual test in the base test class.

### CODE REVIEW

**Comment by Priya91** [comment]
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Comment by Priya91** [comment]
@dotnet-bot test UWP NETNative x86 Release Build

**Comment by davidsh** [comment]
@dotnet-bot Test Outerloop Linux x64 Release Build
@dotnet-bot Test Outerloop NETFX x86 Release Build
@dotnet-bot Test Outerloop UWP CoreCLR x64 Debug Build
@dotnet-bot Test Outerloop Windows x64 Debug Build

File: src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs [hunk 1]
```diff
@@ -292,4 +292,11 @@ public bool IsBypassed(Uri host)
             }
         }        
     }
+
+    public sealed class ManagedHandler_DefaultCredentialsTest : DefaultCredentialsTest, IDisposable
+    {
+        public ManagedHandler_DefaultCredentialsTest(ITestOutputHelper output) : base(output) => ManagedHandlerTestHelpers.SetEnvVar();
+        public void Dispose() => ManagedHandlerTestHelpers.RemoveEnvVar();
+    }
```
Comment (stephentoub) [inline src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs]: Why did you move this to this file? We should be consistent, and currently all of the derived ManagedHandler tests are in the other file. This is now inconsistent.

**Comment by Priya91** [comment]
@stephentoub The test class this managedhandler test is deriving from is compiled only for windows. So it is either this, or creating another file managedhandler.windows.cs and including this test class there.

**Comment by stephentoub** [comment]
> is compiled only for windows

How come?  Any reason we don't just make the tests that are platform specific [PlatformSpecific(...)] instead?

**Comment by Priya91** [comment]
I'm not sure why they are this way, maybe they were written when platformspecific was not avaialble.git sta
